addons/website/static/src/components/views/page_list.xml

```diff
@@ -36,4 +39,27 @@
     </div>
 </t>
 
+<t t-name="website.ActiveWebsiteFacet">
+    <t t-if="state.activeWebsite">
+        <div class="o_searchview_facet o_facet_active_website position-relative d-inline-flex align-items-stretch rounded-2 bg-200 text-nowrap opacity-trigger-hover"
+            role="listitem"
+            aria-label="search"
+            tabindex="0"
+        >
```

Comment:
I'll remove these classes...I was just testing it out. Since other facets don't have the truncate feature let's not add to this one too. Let me know if u think otherwise.

Reply:
better to remove for now. we'll think about it in future.